client/src/components/Ticketing/ticketingmanager/AdminPurchase/AdminCart.tsx

```diff
@@ -0,0 +1,72 @@
+import React, {ReactElement, useState, useEffect} from 'react';
+import {useLocation, useNavigate} from 'react-router';
+import {EventRow} from './AdminPurchase';
+
+/**
+ * Math to dollar - `$${(Math.round(x * 100) / 100).toFixed(2)}`
+ *
+ * @param x
+ */
+const toDollar = (x: number) => `$${(Math.floor(x * 100) / 100).toFixed(2)}`;
+
+interface AdminCartProps {
+  backButtonRoute: string;
+  eventDataFromPurchase: EventRow[];
+}
+/**
+ *
+ * @param AdminCartProps
+ * @param AdminCartProps.backButtonRoute
+ * @param AdminCartProps.eventDataFromPurchase
+ * @returns {ReactElement}
+ */
+const AdminCart = ({
+  backButtonRoute,
+  eventDataFromPurchase,
+}: AdminCartProps): ReactElement => {
+  const location = useLocation();
+  const navigate = useNavigate();
+  const cartItems = location.state?.cartItems || [];
+  const itemsInCart = cartItems.map((item, index) => (
+    <div
+      key={index}
+      className='flex flex-row gap-8 bg-gradient-to-b from-zinc-700 px-5 py-3 rounded-xl mb-5'
+    >
+      <div className='flex-auto text-left'>
+        {item.qty} x {item.name}
+      </div>
+      <div className='flex-auto text-right'>{toDollar(item.price)}</div>
+    </div>
```

Comment:
To give the order items more semantic meaning, I would recommend making the outer div a `p` element with two `span`s within rather than 2 divs. This signals to any screen-reader, as well as just browsers in general, that this is textual information.

Reply:
replaced div's with p and span tags. tested no issues.
![image](https://github.com/WonderTix/WonderTix/assets/97830813/1ae27335-2527-49a1-9b4c-8ebe597f02fd)